pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
 	"Auke Willem Oosterhoff <aukewillem.oosterhoff@mobilityhouse.com>"
```

Comment:
Not being pretentious, but would be nice to mention my name as well

Reply:
Sure, of course! I was thinking to add all addresses that where in the AUTHORS file in the closed-source version of this repo. What do you think?

Reply:
Yeah, sounds good ;)